main.go

```diff
@@ -35,6 +35,9 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
```

Comment:
nit: This should be combined with the import group above.

```suggestion
```

Reply:
fixed with f04e72e